extractor/src/extract/extractor.py

```diff
@@ -13,19 +21,28 @@
 from arkhn_monitoring import Timer
 
 
+def handle_between_filter(col, value):
+    values = value.split(",")
+    if len(values) != 2:
+        raise ValueError("BETWEEN filter expects 2 values separated by a comma.")
+    min_val = values[0].strip()
+    max_val = values[1].strip()
+    return and_(col.__ge__(min_val), col.__le__(max_val))
```

Comment:
have you tried with dates (with maybe different formats) as well ? I think this will be a major use-case of the BETWEEN filter

Reply:
I think we can only filter with strings right now... I was going to open an issue for that